packages/plugin-relay/src/types.ts

```diff
@@ -65,6 +65,10 @@ export type RelayPluginOptions<Types extends SchemaTypes> = EmptyToOptional<{
           resolveNode: (id: { id: string; typename: string }) => MaybePromise<unknown>,
         ) => MaybePromise<unknown>;
       });
+  nodeQueryIdArgOptions: Omit<
+    GlobalIDInputFieldOptions<Types, true, 'Arg', ObjectParam<Types>>,
+    'extensions' | 'required'
```

Comment:
I think it would be better to allow extensions, and just merge them in

Reply:
Agreed, the extensions are merged in now as well 😄.